src/diffusers/pipelines/paint_by_example/pipeline_paint_by_example.py

```diff
@@ -443,8 +443,7 @@ def __call__(
             (nsfw) content, according to the `safety_checker`.
         """
         # 0. Default height and width to unet
-        height = height or self.unet.config.sample_size * self.vae_scale_factor
-        width = width or self.unet.config.sample_size * self.vae_scale_factor
+        width, height = image.size
```

Comment:
This makes sense! But note that `image` can also be a list of PIL image in which case this fails